src/main/java/de/mediathekview/mserver/crawler/sr/tasks/SrTopicsOverviewPageTask.java

```diff
@@ -0,0 +1,99 @@
+package de.mediathekview.mserver.crawler.sr.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.base.utils.UrlParseException;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.sr.SrConstants;
+import de.mediathekview.mserver.crawler.sr.SrTopicUrlDTO;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class SrTopicsOverviewPageTask implements Callable<ConcurrentLinkedQueue<SrTopicUrlDTO>> {
+
+  private static final Logger LOG = LogManager.getLogger(SrTopicsOverviewPageTask.class);
+  
+  private static final String URL_PARAMETER_SUBPAGE = "a_z";
+  private static final String URL_PARAMETER_SHOW_SHORTNAME = "sen";
+  
+  private static final String SHOW_PAGE_URL_SELECTOR = "div.container > div.row > a";
+  private static final String SHOW_LINK_SELECTOR = "h3.teaser__text__header a";
+  
+  @Override
+  public ConcurrentLinkedQueue<SrTopicUrlDTO> call() throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)